### ISSUE TITLE
Do we have a way of dealing with red markings?

### ISSUE THREAD

**Issue by bganglia** [ieee8023/covid-chestxray-dataset#26]

**Comment by bganglia** [comment]
If images from real patients are being compared to these images, the small figure labels (e.g. "A", "B") could also lead to data leakage.

**Comment by ieee8023** [comment]
True. This is a challenge to overcome. However, the models trained with a lot of data don't suffer from this issue though. Look at this example processed using a model trained on the 100k NIH examples:
<img width="949" alt="Screen Shot 2020-03-17 at 9 12 34 PM" src="https://user-images.githubusercontent.com/446367/76915539-0e255e80-6894-11ea-917b-2f3d261814a5.png">

The gradient of the prediction with respect to the input is not using them to make a prediction. So it is possible that the features from those pretrained models (in the torchxrayvision library) can easily ignore the arrows and focus on the right features.

**Comment by bfreskura** [comment]
Is it possible to mark such images in the `metadata.csv`?
